hail/python/hailtop/utils/utils.py

```diff
@@ -1,4 +1,5 @@
-from typing import Callable, TypeVar, Awaitable, Optional, Type, List, Dict, Iterable, Tuple
+from email.generator import Generator
+from typing import Any, Callable, TypeVar, Awaitable, Optional, Type, List, Dict, Iterable, Tuple
```

Comment:
The first import is not needed. We want the Generator type from `typing`. You can just add it in Alphabetical order to the line below.